--- a/Sources/PerceptionCore/Bindable.swift
+++ b/Sources/PerceptionCore/Bindable.swift
@@ -5,10 +5,10 @@
   /// perceptible objects.
   ///
   /// A backport of SwiftUI's `Bindable` property wrapper.
-  @available(iOS, introduced: 13, obsoleted: 17, message: "Use @Bindable without the 'Perception.' prefix.")
-  @available(macOS, introduced: 10.15, obsoleted: 14, message: "Use @Bindable without the 'Perception.' prefix.")
-  @available(tvOS, introduced: 13, obsoleted: 17, message: "Use @Bindable without the 'Perception.' prefix.")
-  @available(watchOS, introduced: 6, obsoleted: 10, message: "Use @Bindable without the 'Perception.' prefix.")
+  @available(iOS, introduced: 13, deprecated: 17, message: "Use @Bindable without the 'Perception.' prefix.")
+  @available(macOS, introduced: 10.15, deprecated: 14, message: "Use @Bindable without the 'Perception.' prefix.")
+  @available(tvOS, introduced: 13, deprecated: 17, message: "Use @Bindable without the 'Perception.' prefix.")
+  @available(watchOS, introduced: 6, deprecated: 10, message: "Use @Bindable without the 'Perception.' prefix.")
   @available(visionOS, unavailable)
   @dynamicMemberLookup
   @propertyWrapper
