--- a/Sources/ComposableArchitecture/Observation/Store+Observation.swift
+++ b/Sources/ComposableArchitecture/Observation/Store+Observation.swift
@@ -311,10 +311,10 @@
   }
 }

-@available(iOS, introduced: 13, obsoleted: 17)
-@available(macOS, introduced: 10.15, obsoleted: 14)
-@available(tvOS, introduced: 13, obsoleted: 17)
-@available(watchOS, introduced: 6, obsoleted: 10)
+@available(iOS, introduced: 13, deprecated: 17)
+@available(macOS, introduced: 10.15, deprecated: 14)
+@available(tvOS, introduced: 13, deprecated: 17)
+@available(watchOS, introduced: 6, deprecated: 10)
 extension Perception.Bindable {
   /// Scopes the binding of a store to a binding of an optional presentation store.
   ///
